requirements.txt

```diff
@@ -1,3 +1,6 @@
-#numpy>=1.10.4
-numpy>=1.7.1
-root_numpy>=4.7.2
+numpy<1.12; python_version == '2.6'
+numpy<1.12; python_version == '3.2'
+numpy<1.12; python_version == '3.3'
+numpy<1.15; python_version == '2.7'
+numpy; python_version >= '3.4'
+# root_numpy ## currently conflicts if numpy isn't installed, as it will download it's own version due to steup_requries
```

Comment:
Who is this installed by then?  The only place I see this being installed presently is `.ci/test_on_docker.sh`.

Reply:
I wasn't able to find a clean solution, so will open an issue for someone who wants to try to tackle it.
Production/development machines will have both installed in some centrally administered way

Reply:
For creating a `virtualenv` on lxplus to allow analysis of calibration data I found the following hack was the only way forward:

https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/b8b6a2ea1c90c0e5162f218d1ff3f75b011abb12/setup/createVirtualEnvPy27.sh#L23-L33

It's not elegant but it does work.  But yeah `root_numpy`'s standard installation was not straight forward.

Reply:
Basically it goes as:
- the python env has `root_numpy` installed with the wrong `numpy` or `ROOT` version,
- you git clone `root_numpy` from it's repo
- install it manually after activating a python `virtualenv`

This was successful although not "elegant"